### PR TITLE
Update Copy and Add Examples #15

### PR DESCRIPTION
**Pull request by nelsonic**
+ [x] Add examples #15

### CODE REVIEW

**Comment by nelsonic** [comment]
Next: https://youtu.be/OLjyCy-7U2U

File: README.md [hunk 1]
```diff
@@ -1,76 +1,144 @@
...
+an Object Oriented Progrramming (OOP) language.
+It supports a varied range of programming aids like interfaces,
+classes, collections, generics, and optional typing.
 
+`Dart` was created by Lars Bak and Kasper Lund
+while working for Google.
+`Dart` is approved as a standard by
+the European Computer Manufacturers Association
+[ECMA 408](https://www.ecma-international.org/publications/standards/Ecma-408.htm).</br>
+The language syntax and semantics are stable don't change.
```
Comment (SimonLab) [inline README.md:36]: Not sure if you want to keep `don't change` or to add `and`:
```suggestion
The language syntax and semantics are stable and don't change.
```

Comment (nelsonic) [reply README.md]: Still very much "Work In Progress" 😉

**Comment by nelsonic** [comment]
Continue: 
+ https://github.com/dwyl/learn-dart/tree/add-examples-issue%2315
+ https://gist.github.com/nelsonic

**Comment by nelsonic** [comment]
@LuchoTurtle could you please take a look at this PR and see what is still relevant while you are learning `Dart`? 🙏

**Pull request merged**